# Conclusions

The genetic cause of this effect may lie further up the immune response pathway, related to IFN production, cytokine response control (SOCS) or macrophages. The increased DGE in asthmatic tissue may be more related to antiviral immune system overreaction than asthmatic response; non-viral assailants probably wouldn’t cause DGE in these genes: this should be tested experimentally with non-viral asthma triggers.

OAS proteins provide a yet-to-be understood antiviral function, independent of RNaseL. This is an important path for therapeutic exploration, potentially providing suppression of the inflammatory response and apoptotic self-destruction, whilst allowing the alternative antiviral pathway to continue functioning.

Inhibitors could be used to suppress, fine-tune or otherwise modify the immune response. Intervening anywhere in the STAT2 & OAS cascade could be beneficial in managing asthma, considering the OAS-RNaseL pathway and the harm its RNA detritus causes. Unchecked, OAS-RNaseL effects may become a positive feedback loop, snowballing a minor reaction into an asthma attack and also initiating development of tissue hypersensitivity and airway remodelling.

This research is significant because it highlights the increased exacerbation of the immune system in asthmatics during response to viral challenge. It draws together knowledge from many previous experimental studies and reviews, revealing potential pathways for therapeutic intervention. It appears to support existing knowledge from the literature and other experimental reports.

## Summary

In summary, STAT2 & OAS genes may not be part of the asthmatic genotype, but their upregulated presence in antiviral inflammatory response offer targets for therapeutic intervention in asthmatics & non-asthmatics alike. Further exploration into polymorphisms and upstream pathway components is required to discover the origins of these differential genetic expressions.

&#x20;

## Further work

* Comparing responses of non‑asthmatic tissues with asthmatic to other pathogens and non-pathogenic triggers (particulates, chemicals, etc) will clarify what trigger classes cause these DGEs.
* Looking for expression correlation between cytokine, antiviral genes and SOCS .
* Extending the statistic tests to cover a broader range of genes (GWAS).
* Exploring the RNaseL-independent antiviral OAS pathway.
* Are any SOCS genes downregulated or differentially expressed in this experimental dataset?
